.github/workflows/publish_draft_releases.yml

```diff
@@ -815,46 +815,9 @@ jobs:
       - name: test - calamari alice peered successfully
         run: |
           grep '\[Parachain\] 💤 Idle (${{ matrix.chain-spec.expected.peer-count.para }} peers)' ${{ github.workspace }}/polkadot-launch/9921.log
-  build-changelog:
-    runs-on: ubuntu-20.04
-    needs: [get-rust-versions, build-runtimes]
```

Comment:
Omg, removing code from this file feels so nice, even if I'm not the one doing :D